lib/huginn_agent/templates/newagent/spec/spec_helper.rb.tt

```diff
@@ -0,0 +1,7 @@
+base_path = File.expand_path('../../', __FILE__)
```

Comment:
I don't fully understand why this is needed, doesn't running `bundle exec rspec  --pattern ../*_spec.rb` work without it?